### DISCUSSION TITLE
NFC icon

### DISCUSSION THREAD

**Discussion by harrybawsac** [tailwindlabs/heroicons#714] [Ideas]
Hey :wave: 

I love using Heroicons for my projects, but I noticed something was missing from the icon library: an NFC icon. Although there are many different NFC icons out there, I think the one used by [the NFC forum](https://nfc-forum.org) is the official/original.

Would like to hear your thoughts on this!

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose

**Reply by harrybawsac** [reply]
Could a mod ban this spammer? @simonswiss

**Reply by zefir-git** [reply]
perhaps also ban everyone asking for social media/brand icons. Anyways, I believe this is what you're looking for:
![image](https://github.com/user-attachments/assets/3d610d6d-fac5-45b9-a460-3acf9b38136b)

**Reply by harrybawsac** [reply]
How is NFC a brand or social media icon? It's a protocol.

**Reply by zefir-git** [reply]
> How is NFC a brand or social media icon? It's a protocol.

Not only is it a brand, it's also a trademark of NFC Forum, Inc.

I highly recommend that you read the following before using it:
https://nfc-forum.org/build/branding
https://www.cognitoforms.com/NFCForum/LicenseAgreementNMark

Are you referring to some other symbol perhaps? At least for me, this is the one I associate with NFC.

And to clarify, Simple Icons isn't just for social media logos, it's for any well-known brand.

**Reply by harrybawsac** [reply]
Alright, mister shill.
